packages/ui-extensions/docs/surfaces/point-of-sale/staticPages/pages/versions.doc.ts

```diff
@@ -13,6 +13,20 @@ const data: LandingTemplateSchema = {
   mobileImage: '/assets/landing-pages/templated-apis/hero.png',
   mobileDarkImage: '/assets/landing-pages/templated-apis/hero-dark.png',
   sections: [
+    {
+      type: 'Generic',
+      anchorLink: 'support-policy',
+      title: 'Support Policy',
+      sectionContent: '',
+      sectionNotice: [
+        {
+          title: 'End of Support Plan',
+          type: 'Warning',
+          sectionContent:
+            'Starting in April 2025, we will end support for versions on a one-year rolling basis. Support for legacy versions (1.0.0 - 1.7.0) and 2024-04 will end on the next POS release after April 1, 2025. Then, support for 2024-07 will end on the next POS release after July 1, 2025, and so on. Refer to the [migration guide](/docs/api/pos-ui-extensions/migrating) for more information.',
```

Comment:
Support for all legacy versions older than and including 2024-04 (1.0.0, 1.1.2, 1.2.0, 1.3.0, 1.4.0, 1.5.0, 1.6.0, 1.7.0, 2024.04) will be removed on the 2025.04 (April 1, 2025) release. On the 2025.07 (July 1, 2025 release), support for 2024.07 will be removed, and so on.

^ can we change to this to be SUPER clear and precise?